tests/phpunit/tests/test-admin.php

```diff
@@ -49,6 +49,25 @@ public function test_admin_bar_menu() {
 		$this->assertEquals( '/wp-admin/edit.php?lang=fr', $fr->href );
 	}
 
+	public function test_admin_bar_menu_should_hide() {
+		global $wp_admin_bar;
+		add_filter( 'show_admin_bar', '__return_true' ); // Make sure to show admin bar.
+
+		$this->go_to( admin_url( 'post-new.php?post_type=page' ) );
+		$GLOBALS['pagenow'] = 'post-new.php';
+		$GLOBALS['typenow'] = 'page';
```

Comment:
Maybe add cleanup for this in `tearDown()` ?

Reply:
This is done [here](https://github.com/polylang/polylang/blob/480fb6cb006679dc87f6ad8fb33708f318700a54/tests/phpunit/includes/testcase-trait.php#L142). Except that `$GLOBALS['typenow']` is missing so I added it in [a1d5e3f](https://github.com/polylang/polylang/pull/1698/commits/a1d5e3f4a356b4d0207def1d02d6798d9bcba998).